lib/intelligent_foods/resources/menu.rb

```diff
@@ -2,36 +2,24 @@
 
 module IntelligentFoods
   class Menu < IntelligentFoods::Object
+    extend IntelligentFoods::ApiOperations::Retrieve
```

Comment:
should this be on the Object class? or is retrieval not applicable to the other resources?